newm-server/src/main/kotlin/io/newm/server/database/migration/V31__ConfigUpdates.kt

```diff
@@ -9,11 +9,9 @@ import org.jetbrains.exposed.sql.transactions.transaction
 class V31__ConfigUpdates : BaseJavaMigration() {
     override fun migrate(context: Context?) {
         transaction {
-            execInBatch(
-                listOf(
-                    // default 24 hours
-                    "INSERT INTO config VALUES ('$CONFIG_KEY_EVEARA_STATUS_CHECK_MINUTES','1440') ON CONFLICT(id) DO NOTHING",
-                )
+            exec(
```

Comment:
@saif-software-developer Revert this change. We don't want to be changing old migrations that have already been run on our databases out there. There's no material difference in this one, but it's more about the principle of it.

Reply:
Actually. This is fine for now. In the future, don't modify past migrations, but I'm going to get this PR merged.